samples/bluetooth/fast_pair/input_device/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -4,29 +4,32 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
-&mram1x {
-	/delete-node/ cpuapp_rw_partitions;
+/delete-node/ &cpuapp_rw_partitions;
 
+&mram1x {
 	cpuapp_rw_partitions: cpuapp-rw-partitions {
 		compatible = "nordic,owned-partitions", "fixed-partitions";
 		status = "okay";
-		perm-read;
-		perm-write;
-		perm-secure;
+		nordic,access = <NRF_OWNER_ID_APPLICATION NRF_PERM_RWS>;
 		#address-cells = <1>;
 		#size-cells = <1>;
 
 		dfu_partition: partition@100000 {
 			reg = <0x100000 DT_SIZE_K(908)>;
 		};
 
-		storage_partition: partition@1e3000 {
-			reg = <0x1e3000 DT_SIZE_K(20)>;
+		/* Align the partition size to 4096 B to be able to enable the
+		 * fprotect in the future.
+		 */
+		bt_fast_pair_partition: partition@1e3000 {
+			reg = <0x1e3000 DT_SIZE_K(4)>;
 		};
 
-		bt_fast_pair_partition: partition@1e8fb8 {
-			label = "bt_fast_pair";
-			reg = <0x1e8fb8 0x48>;
+		/* The storage partition needs to start at address divisible
+		 * by 4096 (0x1000), because this is the storage sector size.
```

Comment:
Actually the storage sector size is not a fix value and can be set by the app, I don't know where you got the information that the storage sector size should be 4096 bytes for the nrf54h20.
For NVS this size should be a multiple of the erase-block-size value that is set in the device tree
For ZMS this size should be a multiple of the write-block-size (I think it is 16 bytes for the nrf54h20).
I am not arguing about the alignment here, as this is necessary for NVS, but the comment is kind of misleading

Reply:
I know it can be set by application, but in our sample it's 4096 so that's what I'm referring to. Also, from what I see the alignment is also needed for ZMS - when I set the storage partition to `reg = <0x1e3080 0x5000>;`, the sample fails to boot. I guess it would be nice to have some build assert for asserting that the storage partition starts at adress divisible by storage sector size (from what I experience this is the silent requirement)

Reply:
ZMS do not need the alignment of the start address to the sector size. It should be aligned only to the write-block-size
If the sample fails to boot, then there is a bug that needs to be reported.

Reply:
So in that case there's bug here. I set start address to 0x1e3080 so it was aligned to write-block-size which is 16 bytes (0x10), but it failed to boot. What is more when I increased the size (`reg = <0x1e3080 0x5000>;` -> reg = <0x1e3080 0x5f80>;) it started working.

Reply:
So @rghaddab, what do you want the comment to look like?